index.js

```diff
@@ -0,0 +1,71 @@
+document.addEventListener("DOMContentLoaded", () => {
```

Comment:
Overall very good performance. All functionalities are working as intended, and styling is followed strictly in accordance with the rubric. I left few feedbacks below to improve the app in the future. Great work and looking forward to see more projects in the future.